2018/g2018_vr.csv

```diff
@@ -4,31 +4,31 @@ AK,2018-10-07,2018-10-07,,,
 AZ,2018-10-09,2018-10-09,,,
 AR,2018-10-07,,,,
 CA,2018-10-22,2018-10-22,,,
-CO,,,,,Voters can register at a polling center up to and including Election Day.
-CT,2018-10-07,2018-10-07,,,
+CO,2018-11-06,,,,Voters can register at a polling center up to and including Election Day.
+CT,2018-10-30,2018-10-07,,,
 DE,2018-10-13,2018-10-13,,,Voters living overseas have until 10/22/2018 to register to vote.
 FL,2018-10-09,2018-10-09,,,
 GA,2018-10-09,2018-10-09,,,
 HI,2018-10-09,2018-10-09,,,
 ID,,,,,
 IL,2018-10-09,,,,"Voters in Illinois can register to vote on any day, including Election Day"
 IN,2018-10-09,2018-10-09,,,
-IA,,,,,"You may register and vote on election day. To do so, you first must go to the correct polling place for your current address on election day."
+IA,2018-10-27,,,,"You may register and vote on election day. To do so, you first must go to the correct polling place for your current address on election day."
 KS,2018-10-16,2018-10-16,,,"Kansas still has 2016 information on their website, so this is out best guess at the moment."
 KY,2018-10-09,2018-10-09,,,
 LA,2018-11-07,2018-11-17,,,Online voter registration deadline is 11/17/2018
 ME,,,,,Voters can register on election day at a local election office or city hall. The deadline to register by mail or through a voter registration drive is the close of business on the 21st day before the election: 2018-10-16
 MD,2018-10-16,2018-10-16,,,
 MA,2018-10-17,2018-10-17,,,
 MI,2018-10-09,,,,
-MN,,,,,"The deadline to register online is 11:59 p.m., 21 days before Election Day. The deadline to register on paper is 5 p.m., 21 days before Election Day. Otherwise, you can register on Election Day at your polling place."
+MN,2018-10-16,2018-10-16,,,"The deadline to register online is 11:59 p.m., 21 days before Election Day. The deadline to register on paper is 5 p.m., 21 days before Election Day. Otherwise, you can register on Election Day at your polling place."
 MS,2018-10-08,,,,10/9/2018 is the deadline to register to vote by mail
 MO,2018-10-10,2018-10-10,,,
 MT,2018-10-09,,,,"Regular voter registration closes at 5 p.m. on 10/9/2018.
 Individuals can late-register at the county election office beginning the next day and through close of polls on election day."
 NE,2018-10-19,2018-10-19,,,
-NV,2018-10-09,2018-10-18,,,In-person deadline is 10/16/2018; online voter registration deadline is 10/18/2018
-NH,,,,,"GENERAL ELECTION DAY- November 6, 2018 - unregistered voters may register and vote on this day."
+NV,2018-10-16,2018-10-18,,,In-person deadline is 10/16/2018; online voter registration deadline is 10/18/2018
+NH,2018-11-06,,,,"GENERAL ELECTION DAY- November 6, 2018 - unregistered voters may register and vote on this day."
```

Comment:
See the [other comment](https://github.com/democrats/election-calendar/pull/36/files#r170463780) and #7